storybook/storybook-react/src/Overlap/Overlap.stories.tsx

```diff
@@ -0,0 +1,49 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { AspectRatio, Grid, Image, Overlap, SearchField } from '@amsterdam/design-system-react'
+import { Meta, StoryObj } from '@storybook/react'
+
+const meta = {
+  title: 'Layout/Overlap',
+  component: Overlap,
+  argTypes: {
+    children: {
+      table: {
+        disable: true,
+      },
+    },
+  },
+} satisfies Meta<typeof Overlap>
+
+export default meta
+
+type Story = StoryObj<typeof meta>
+
+export const Default: Story = {
+  args: {
+    children: (
+      <>
```

Comment:
This fragment isn't necessary if we make `children` an array

Reply:
Yes, updated.